internal/services/projects_test.go

```diff
@@ -114,7 +123,7 @@ func Test_createProject(t *testing.T) {
 			accessManagementWrapper: &mock.AccessManagementMockWrapper{},
 			projectGroups:           "grp1,grp2",
 			featureFlagsWrapper:     &mock.FeatureFlagsMockWrapper{},
-		}, want: "ID-new-project-name", wantErr: true},
+		}, want: "", wantErr: true},
```

Comment:
Why did you change this test case?

Reply:
before the fix, we fetch the groups after creating the project, so also in a case of error we returned projectID
now we're fetching groups before the creation so in case of failure we still don't have projectID and in this case we are getting an empty string